### ISSUE TITLE
Its can use outside of china?

### ISSUE THREAD

**Issue by lordmen99** [zyktojo/AmapSmoothMarker#3]
This demo app,can use worldwide map? or just for in china map

**Comment by zyktojo** [comment]
This is just use for China map app:Amap,
If you want use for Google Map,I will do another one Demo for Google Map~

**Comment by lordmen99** [comment]
yes,please create demo for google map so can show smooth movement marker

**Comment by lordmen99** [comment]
hi, u have create demo with google map?
